packages/faustwp-cli/tsconfig.json

```diff
@@ -1,12 +1,15 @@
 {
-  "compilerOptions": {
-    "strict": true,
-    "outDir": "dist",
-    "module": "ES2015",
-    "target": "ES2017",
-    "esModuleInterop": true,
-    "moduleResolution": "node",
-    "declaration": true
-  },
-  "exclude": ["dist", "./*.d.ts"]
+	"compilerOptions": {
+		"strict": true,
+		"outDir": "dist",
+		"module": "ES2015",
+		"target": "ES2017",
+		"esModuleInterop": true,
+		"moduleResolution": "node",
+		"declaration": true
+	},
```

Comment:
```suggestion
	},
	"include": ["src"],
```

Very important we include the newly created `src` directory in the `tsconfig.json`. Without it, our dist directory includes `src` (i.e. `dist/src/index.js`) and the build will break.

<img width="350" alt="Screenshot 2023-03-22 at 2 09 49 PM" src="https://user-images.githubusercontent.com/5946219/227011806-b4d1cf6b-550b-4f74-9d45-6192eb1601d0.png">